components/support/ffi/src/lib.rs

```diff
@@ -87,20 +88,22 @@
 //!    also use the versions of these functions in the [`abort_on_panic`] module, which
 //!    do as their name suggest.
 //!
-//! Additionally, c strings that are passed in as arguments may be converted to rust strings using
-//! helpers such as [`rust_str_from_c`], [`opt_rust_str_from_c`], [`rust_string_from_c`],
-//! [`opt_rust_string_from_c`], etc.
+//! Additionally, c strings that are passed in as arguments may be represented using [`FfiStr`],
+//! which contains several helpful inherent methods for extracting their data.
 //!
 
 use std::{panic, thread};
 
 mod error;
+mod ffistr;
 pub mod handle_map;
 mod into_ffi;
+#[macro_use]
```

Comment:
I assume this is still needed despite the `crate::macros::*;` import bellow

Reply:
Yeah, seems to be. Not sure what the deal is, tbh